packages/cras.rb

```diff
@@ -66,7 +66,7 @@ def self.build
           --disable-webrtc-apm \
           --enable-sse42 \
           --enable-avx \
-          --enable-avx2 
+          --enable-avx2
```

Comment:
Thanks for your pull request! Can you help us to add the missing backslash in this line? 
```shell
--enable-avx2 \
```

Reply:
We can address this in a separate pull request.  Thank you @kjperkin!